### PR TITLE
proof reading. typos, comments

### PR DESCRIPTION
**Pull request by luisschubert**
enjoy reading the book so far. thanks for sharing this freely!

### CODE REVIEW

File: chapters/power.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ In our 21st century, identity is just one part of the much larger way that we us
...
...s not necessarily mean understanding it, and understanding it does not necessarily mean acting upon it. It is easy to forget this, especially in our modern world, where information technology like Google, the internet, and smartphones clamor for our attention, and seem to be responsible for so much in society. Underlying technology, however, are always human beings interpreting information and acting upon it. But it's also easy to confuse information _technology_ for the cause of this power. Consider, for example, Google search, an inescapable fixture in how the modern world finds information. What is Google search without content? Merely an algorithm that finds and organizes nothing. Or, think about using your smartphone without an internet connection or without anyone to talk to. Of what value is its microprocessor, it's memory, and its applications if there is no content to consume and no people to connect with? Information technology is, ultimately, just a vehicle for information to be stored, retrieved, and presented--there is nothing intrinsically valuable about it. And information itself has the same limited powered: without people to understand and act upon it, it is inert.
```
Comment (luisschubert) [inline chapters/power.md]: "Google" isn't information technology per se. "Google Search" is.

Comment (luisschubert) [inline chapters/power.md]: > It is easy to forget this, especially in our modern world, where information technology like Google, the internet, and smartphones clamor for our attention, and seem to be responsible for so much in society. 

Don't love this sentence.  
What is it exactly responsible for? And what is the IT?

**Comment by luisschubert** [comment]
@amyjko  I didn't intend for this to get merged. Just thought providing proof-reading feedback would be easier to see in form of a PR than creating a bunch of issues for little nit-picky comments.

**Comment by amyjko** [comment]
Thank you for the excellent copyediting!

**Pull request merged**